src/main/java/se/sundsvall/supportmanagement/api/NotificationsResource.java

```diff
@@ -70,12 +71,17 @@ public ResponseEntity<List<Notification>> getNotifications(
 
 	@PostMapping
 	@ApiResponse(responseCode = "201", description = "Created - Successful operation", headers = @Header(name = LOCATION, schema = @Schema(type = "string")), useReturnTypeSchema = true)
+	@ApiResponse(responseCode = "409", description = "Conflict", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
 	@Operation(summary = "Create notification", description = "Create new notification for the namespace and municipality")
 	public ResponseEntity<Void> createNotification(
 		@Parameter(name = "namespace", description = "Namespace", example = "my.namespace") @Pattern(regexp = NAMESPACE_REGEXP, message = NAMESPACE_VALIDATON_MESSAGE) @PathVariable final String namespace,
 		@Parameter(name = "municipalityId", description = "Municipality id", example = "2281") @ValidMunicipalityId @PathVariable final String municipalityId,
 		@Valid @NotNull @RequestBody final Notification notification) {
+
 		final var result = notificationService.createNotification(municipalityId, namespace, notification);
```

Comment:
In case you don't want this extra if-clause:
```
final var result = Optional.ofNullable(notificationService.createNotification(municipalityId, namespace, notification))
.orElseThrow(() -> Problem.valueOf(Status.CONFLICT, "Notification already exists"))
```